x-pack/plugins/security_solution/public/detections/components/rules/eql_query_bar/footer.tsx

```diff
@@ -44,9 +43,11 @@ export interface Props {
 
 type SizeVoidFunc = (newSize: string) => void;
 
-const Container = styled(EuiPanel)`
+const Container = styled(EuiFlexGroup)`
   border-radius: 0;
-  background: ${({ theme }) => theme.eui.euiPageBackgroundColor};
+  border-bottom-left-radius: 6px;
+  border-bottom-right-radius: 6px;
+  background: ${({ theme }) => theme.eui.euiColorLightestShade};
```

Comment:
does this work correctly with the dark theme? 🤔

Reply:
Setting "light" colors was giving problems using the dark theme, it may have been solved from EUI. Thanks for checking!